content/bn/infrastructure-as-a-service.md

```diff
@@ -0,0 +1,23 @@
+---
+title: পরিষেবা হিসেবে পরিকাঠামো (Infrastructure as a service)
+status: Completed
+category: ধারণা
+---
+
+
+## এটা কি
+
+পরিষেবা হিসেবে পরিকাঠামো অথবা IaaS হল একটি [ক্লাউড কম্পিউটিং](bn/cloud-computing/) পরিষেবার আদল যেটি Pay as you go এর আদলে [ফিজিক্যাল](bn/bare-metal-machine/) অথবা [ভার্চুয়ালাইজড](bn/virtualization/)  কম্পিউট, স্টোরেজ এবং প্রয়োজনে নেটওয়ার্ক রিসোর্স প্রদান করে। ক্লাউড প্রদানকারীরা হার্ডওয়্যার এবং সফটওয়্যার এর মালিক হন এবং পরিচালনা করেন যা গ্রাহকদের জন্য সরকারি ,ব্যক্তিগত কিংবা হাইব্রিড ক্লাউড স্থাপনায় উপলব্ধ।                                 
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+প্রথাগতভাবে সংস্থাগুলি প্রায়ই কার্যকর কম্পিউটিং সংস্থানের  সংকুলান নিয়ে সমস্যায় পড়ে | এমনকি যদি এটি শুধুমাত্র 1% সময়ের জন্য প্রয়োজন হয় তবে সম্ভাব্য সর্বোচ্চ চাহিদার জন্য তথ্য কেন্দ্র তৈরি করতে হবে |  কম চাহিদার সময় এই গণনা সংস্থাগুলি নিষ্ক্রিয় থাকে এবং যদি কাজের চাপ প্রত্যাশিত চাপেও ছাড়িয়ে যায় সেই কাজের চাপ প্রক্রিয়া করার জন্য কম্পিউটিং সংস্থানগুলির অভাব রয়েছে | স্কেলেবিলিটির এই অভাবের ফলে খরচ বেড়ে যায় এবং সম্পদের অপ্রয়োজনীয় ব্যবহার হয়।                                                                   
+
+
+
+## এটা কিভাবে সাহায্য করে
+
+IaaS সংস্থাগুলি তাদের ব্যবহারের জন্য গণনা এবং তথ্য কেন্দ্রের স্থান ক্রয় এবং দেখভাল করা বন্ধ রাখতে পারে। প্রয়োজন ভিত্তিক কাঠামো তাদের প্রয়োজন অনুসারে কম্পিউট রিসোর্স ভাড়া করতে এবং মূলধন ব্যয় ব্যাহত করতে অথবা [CAPEX](https://en.wikipedia.org/wiki/Capital_expenditure) পিছিয়ে দেওয়ার অনুমতি দেয়। যখন তাদের স্কেলের ওঠা পড়া হয় সেই অনুসারে। 
```

Comment:
Kindly create a moderate sentence. This seems just copying from Google Translator.

Reply:
Sorry, Sir. I have corrected my mistakes. Please check that whenever you are free.